tests/playwright/pages/wp-admin-page.js

```diff
@@ -101,4 +102,10 @@ module.exports = class WpAdminPage extends BasePage {
 		await this.page.locator( '.elementor-tab-control-' + panelName + ' a' ).click();
 		await this.page.waitForSelector( '.elementor-tab-control-' + panelName + '.elementor-active' );
 	}
+
+	async setLanguage( language ) {
+		await this.page.goto( '/wp-admin/options-general.php' );
+		await this.page.selectOption( '#WPLANG', language );
+		await this.page.locator( '#submit' ).click();
+	}
```

Comment:
For my PR I have changed this method a bit. 

I have defined the parameters. 
And I have added the option to change the language back to English, which needs some additional work, because the default language is listed with `value=""` inside the dropdown.

/**
	 * Set a different language in the WordPress settings.
	 *
	 * @param {string} language - The language code, default `he_IL`;
	 *
	 * @return {Promise<void>}
	 */
	async setLanguage( language = 'he_IL' ) {
		await this.page.goto( '/wp-admin/options-general.php' );

		let languageCheck = language;

		if ( 'he_IL' === language ) {
			languageCheck = 'he-IL';
		}

		// Check if the language is active already.
		if ( ! await this.page.$( 'html[lang=' + languageCheck + ']' ) ) {
			// Extra workaround for the selection of the default language.
			if ( 'en_US' === language && ! await this.page.$( '#WPLANG option[value=en_US]' ) ) {
				language = '';
			}

			await this.page.selectOption( '#WPLANG', language );
			await this.page.locator( '#submit' ).click();
		}
	}

Reply:
we can use the default !